installers/method/kubectl/client-setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#  Copyright 2020 Crunchy Data Solutions, Inc.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This script should be run after the operator has been deployed
+export PGO_NAMESPACE=${PGO_NAMESPACE:-pgo}
+
+# Check that the pgo.tls secret exists
```

Comment:
I think you mean `pguser-admin` secret and not `pgo.tls`?

Reply:
yup